decoders/ps2/__init__.py

```diff
@@ -18,9 +19,11 @@
 ##
 
 '''
-This protocol decoder can decode PS/2 device -> host communication.
+This protocol decoder can decode PS/2 device -> host communication \
```

Comment:
just noticed - are you sure you need the trailing \  ? see e.g. the uart PD

Reply:
Yeah. I don't actually want to force a line break there, but I also don't want to exceed the standard column width (80 characters)